### PR TITLE
feat: add ruby run events pubsub sample

### PR DESCRIPTION
**Pull request by grant**
Adds a simple Ruby sample for Run events w/ Pub/Sub
- Uses alpha commands for consistency

### CODE REVIEW

**Review by TheRoyalTnetennba — APPROVED** [review]

File: run/events-pubsub/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Cloud Run Hello World Sample
+
+This sample shows how to deploy a Hello World application to Cloud Run.
```
Comment (averikitsch) [inline run/events-pubsub/README.md:3]: Needs updating

File: run/events-pubsub/app.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+post "/" do
+  request.body.rewind # in case someone already read it
+
+  body = JSON.parse request.body.read
+  data = Base64.decode64 body["message"]["data"]
+  if data.empty?
+    data = "World"
+  end
+
+  if request.has_header?('ce-id')
```
Comment (averikitsch) [inline run/events-pubsub/app.rb:33]: nit: I would use a ternary operator here

File: run/events-pubsub/app.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    id = request.get_header('ce-id')
+  else
+    id = ''
+  end
+
+  result = "Hello " + data + "! ID: " + id
```
Comment (averikitsch) [inline run/events-pubsub/app.rb:39]: nit: use string interpolation
```suggestion
  result = "Hello #{data}! ID: #{id}"
```

**Review by averikitsch — CHANGES_REQUESTED** [review]

**Comment by grant** [comment]
Thanks for review. PTAL.

Tested locally and on Cloud Run.

File: run/events-pubsub/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+--target-service run-events-pubsub \
+--type com.google.cloud.pubsub.topic.publish \
+--parameters topic=my-topic
+```
+
+Deploy your Cloud Run service:
+
+```
+gcloud builds submit \
+ --tag gcr.io/$(gcloud config get-value project)/run-events-pubsub
```
Comment (averikitsch) [inline run/events-pubsub/README.md:31]: Looks like other languages are using `cloudrun-events-pubsub` as the container name

**Review by averikitsch — APPROVED** [review]

File: run/events-pubsub/README.md (hunk 4 above)
Comment (grant) [reply run/events-pubsub/README.md:31]: Sounds good

**Comment by grant** [comment]
@TheRoyalTnetennba Merge this PR?

**Pull request merged**